libobs/obs.h

```diff
@@ -2105,6 +2105,17 @@ EXPORT size_t obs_output_get_mixers(const obs_output_t *output);
 EXPORT void obs_output_set_video_encoder(obs_output_t *output,
 					 obs_encoder_t *encoder);
 
+/**
+ * Sets the current video encoder associated with this output,
+ * required for encoded outputs
```

Comment:
Use ending punctuation if comment is more than a single sentence. See the comment for `obs_output_set_audio_encoder` below.

Fix throughout.